dlp/snippets/deid/deid_test.go

```diff
@@ -117,3 +119,54 @@ func TestDeidentifyExceptionList(t *testing.T) {
 	}
 
 }
+
+func TestDeidentifyTableBucketing(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	row1 := &dlppb.Table_Row{
+		Values: []*dlppb.Value{
+			{Type: &dlppb.Value_StringValue{StringValue: "22"}},
+			{Type: &dlppb.Value_StringValue{StringValue: "Jane Austen"}},
+			{Type: &dlppb.Value_StringValue{StringValue: "21"}},
+		},
+	}
+
+	row2 := &dlppb.Table_Row{
+		Values: []*dlppb.Value{
+			{Type: &dlppb.Value_StringValue{StringValue: "55"}},
+			{Type: &dlppb.Value_StringValue{StringValue: "Mark Twain"}},
+			{Type: &dlppb.Value_StringValue{StringValue: "75"}},
+		},
+	}
+
+	row3 := &dlppb.Table_Row{
+		Values: []*dlppb.Value{
+			{Type: &dlppb.Value_StringValue{StringValue: "101"}},
+			{Type: &dlppb.Value_StringValue{StringValue: "Charles Dickens"}},
+			{Type: &dlppb.Value_StringValue{StringValue: "95"}},
+		},
+	}
+
+	table := &dlppb.Table{
+		Headers: []*dlppb.FieldId{
+			{Name: "AGE"},
+			{Name: "PATIENT"},
+			{Name: "HAPPINESS SCORE"},
+		},
+		Rows: []*dlppb.Table_Row{
+			{Values: row1.Values},
+			{Values: row2.Values},
+			{Values: row3.Values},
+		},
+	}
+
+	buf := new(bytes.Buffer)
+	if err := deIdentifyTableBucketing(buf, tc.ProjectID, table); err != nil {
+		t.Errorf("deIdentifyTableBucketing: %v", err)
+	}
+	got := buf.String()
+	if want := "Table after de-identification"; !strings.Contains(got, want) {
+		t.Errorf("deIdentifyTableBucketing got %q, want %q", got, want)
+	}
```

Comment:
Please see the Java example. And corresponding test dlp.snippets.DeIdentificationTests#testDeIdentifyTableBucketing in Java samples repo.
Please confirm that the bucketing is actually happening.

Reply:
Hii @annadata, 
Thank you for your suggestion and guidence, As you have suggested I have made the changes as per the java-code samples currently, just want to point out one thing for your reference that will not be aligned with existing go-code samples: 
- You can see that any go code sample has not any other return type except `error`. but for this sample's test case I have to compare with `expectedTable` , and for that, I have to provide return type `*dlppb.Table` alongside with error.

Reply:
@AarshD-crest 
Sorry for not being clear. Please confirm that "75" in input is replaced with ""70:80".

Reply:
Hiii @annadata , I have updated the test case as per your suggestion. tell me if anything more is required.